subprojects/gradle/runner/service/src/main/kotlin/com/avito/runner/service/worker/device/adb/AdbDevicesManager.kt

```diff
@@ -36,26 +36,16 @@ class AdbDevicesManager(
                 shouldRetry
             }
             .map { output ->
-                output
-                    .substringAfter("List of devices attached")
-                    .split(System.lineSeparator())
-                    .asSequence()
-                    .map { it.trim() }
-                    .filter { it.isNotEmpty() }
-                    .filter { it.contains("online") || it.contains("device") }
-                    .map {
+                AdbDeviceParser().parse(output)
```

Comment:
class has no state, why bother? kotlin has object or top-level function for this case

Reply:
my habits from android development - don't make objects kinda static